src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/GetKnownPackagesFromAssetManifests.cs

```diff
@@ -32,7 +32,14 @@ public override bool Execute()
 
             KnownPackages = xDocuments
                 .SelectMany(doc => doc.Root!.Descendants("Package"))
-                .Select(package => new TaskItem(package.Attribute("Id")!.Value, new Dictionary<string, string> { { "Version", package.Attribute("Version")!.Value } }))
+                .Select(package =>
+                    new TaskItem(
+                        package.Attribute("Id")!.Value,
+                        new Dictionary<string, string>
+                        {
+                            { "Version", package.Attribute("Version")!.Value },
+                            { "DotNetReleaseShipping", package.Attribute("DotNetReleaseShipping")?.Value ?? "false" }
+                        }))
```

Comment:
Please elaborate on this change. I'm not sure why this PR needs that.

Reply:
This was added to enable downstream targets (like the one I added) to determine if an asset is shipping vs non-shipping. This was what allowed the target to determine the non-shipping version to use for the sentinel package even if no other non-shipping package is used.

I'll try an alternative approach.